src/core/BufferGeometry.js

```diff
@@ -56,7 +56,7 @@ THREE.BufferGeometry.prototype = {
 
 		if ( name === 'index' ) {
 
-			console.error( 'THREE.BufferGeometry.addAttribute: Use .setIndex() for index attribute.' );
+			console.warn( 'THREE.BufferGeometry.addAttribute: Use .setIndex() for index attribute.' );
```

Comment:
Why are you doing this? I think it's no longer supported - warning typically means we correct the problem, error means it's fatal. A missing index buffer sure is.